src/components/CalibrationManagerCells.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+    ModalBody, 
```

Comment:
fix all indentation in this file, 4 => 2 spaces per tab

Reply:
oh no i just changed my editor default space from 4 to 2!